lib/src/wait.dart

```diff
@@ -3,6 +3,8 @@ import 'dart:async';
 import 'booleans.dart';
 
 /// Wait Helper Utilities
+///
+/// `since 0.4.0`
```

Comment:
Might be a good idea to add putting these `since` tags into the PR checklist template?